test/integration/runner_task_test.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+require 'test_helper'
+
+require 'kubernetes-deploy/runner_task'
+class RunnerTaskTest < KubernetesDeploy::IntegrationTest
+  def test_works
+    deploy_fixtures("hello-cloud")
+
+    task_runner = KubernetesDeploy::RunnerTask.new(
+      context: KubeclientHelper::MINIKUBE_CONTEXT,
+      namespace: @namespace,
+    )
+
+    task_runner.run(
+      task_template: 'hello-cloud-template-runner',
+      command: nil, # idk
+      args: ['a']
+    )
+
+    assert_logs_match(/Starting task runner/)
```

Comment:
These tests are expensive, so I'd suggest doing one big test for the success case, that checks all the expected logs, and the command, and the args, etc.